test/rest_api/conftest.py

```diff
@@ -18,6 +18,9 @@
 from cassandra.cluster import Cluster, ConsistencyLevel, ExecutionProfile, EXEC_PROFILE_DEFAULT
 from cassandra.policies import RoundRobinPolicy
 
+sys.path.insert(1, sys.path[0] + '/../..')
```

Comment:
See above

Reply:
Unfortunately, here the situation is the same.